ted_sws/core/entrypoints/api/main.py

```diff
@@ -0,0 +1,124 @@
+import hashlib
+import logging
+import subprocess
+import uuid
+from enum import Enum
+from typing import Optional, Union, Tuple
+
+import click
+from fastapi import FastAPI
+
+from ted_sws.event_manager.adapters.logger import Logger, LOG_INFO_LEVEL
+from ted_sws.event_manager.domain.message_bus import message_bus
+from ted_sws.event_manager.model.message import Log
+
+
+class ResponseType(Enum):
```

Comment:
these shall be moved into a project component, called  `identity_manager` or something like that.